### DISCUSSION TITLE
[RNW] Getting error when using Animated.Value for style prop

### DISCUSSION THREAD

**Discussion by matallui** [nativewind/nativewind#259] [Q&A] [answered]
Hi! I have this Expo project which runs on native and web, but I've started getting some errors when I try to use `Animated.Value` as a `style` prop.
However, I noticed if I add the following to my `App.tsx` it starts working fine, so this seems to only occur when I use CSS for web:
```tsx
NativeWindStyleSheet.setOutput({
  default: "native",
});
```

An example snippet of what I'm doing:
```tsx
const Component = () => {
  const fontSize = useRef(new Animated.Value(24)).current;

  return (
    <View className='flex-1 other-styles'>
      <Animated.Text style={{fontSize}} className='font-bold other-styles'>Hello World</Animated.Text>
    </View>
  )
}
```

When I run this code on Web, even though everything seems to work fine, I keep getting these errors constantly:

```
styleq: fontSize typeof 24 is not "string".
```

I tried to reproduce this on a new project but didn't see the error.
I've also just replaced the contents of my current project's `App.tsx` to include a basic `View` and `Animated.Text` and I can't replicate the issue either.
It seems that this only occurs when I'm rendering the components inside a navigation screen or something.

Any idea what this could be? Or how I could fix it?

I guess worst case I can always use `default: "native"`, but was trying to figure out a better way.

**Comment by marklawlor** [comment, marked as answer]
RNW doesn't allow mixing the two style types (CSS and inline) on animated components, but a fix is currently being tested in 0.19 https://github.com/necolas/react-native-web/pull/2377

**Reply by matallui** [reply]
Awesome, thanks for the response! Maybe we'll just put up with the errors or switch to native styles until this gets fixed.

**Reply by matallui** [reply]
@marklawlor btw when I switch NativeWind's output to `native` I noticed the `hover:` classes don't work as expected. When I hover over the elements it gets activated, but never goes away when I hover out. Is this an issue or is this just nor supported with `native` output?

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
